docs/codeql/ql-training/query-examples/java/data-flow-java-1.ql

```diff
@@ -4,7 +4,7 @@ class StringConcat extends AddExpr {
   StringConcat() { getType() instanceof TypeString }
 }
 
-from MethodAccess ma
+from MethodCall ma
```

Comment:
Should MethodCall variables named `ma` be renamed to `mc`?

Reply:
Ideally yes, but it's a harder transformation to script!